contracts/TreasuryActions.cdc

```diff
@@ -265,7 +267,9 @@ pub contract TreasuryActions {
 
     init(_signer: Address) {
       self.signer = _signer
-      self.intent = "Remove ".concat((_signer as Address).toString()).concat(" as a signer to the Treasury.")
+      self.intent = "Remove "
+                      .concat((_signer as Address).toString())
```

Comment:
we can probably remove the `as Address` casting to make it consistent with the AddSignerAction